lib/src/main/java/com/basistheory/android/view/mask/Mask.kt

```diff
@@ -0,0 +1,76 @@
+package com.basistheory.android.view.mask
+
+internal class Mask(
+    private val mask: List<Any>,
+) {
+    private val maskPattern =
+        convertMaskPatternToString(
+            mask,
+            "_"
```

Comment:
shouldn't this default placeholder be configurable?

Reply:
It isn't displayed at the moment, I guess it would be useful if the user wanted it to be displayed. 🤔